src/App.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
-import logo from './logo.svg';
```

Comment:
Total nitpick but we could consider 🔪 removing the React logo file as we'll never use it. 🧹